src/main/java/seedu/address/model/ingredient/Ingredient.java

```diff
@@ -2,27 +2,29 @@
 
 import java.util.Objects;
 
+import seedu.address.model.Item;
+
 /**
  * Represents an ingredient in the book.
  * Guarantees: details are present and not null, field values are validated, immutable.
  */
 
-public class Ingredient {
+public class Ingredient implements Item {
     // Identity fields
-    private final String ingredient;
+    private final String ingredientName;
     private int unit;
 
     /**
      * Every field must be present and not null.
      */
     public Ingredient(String ingredientName, int ingredientUnit) {
-        this.ingredient = ingredientName;
+        this.ingredientName = ingredientName;
```

Comment:
Might want to enforce a non-negative check here (throw some exception if ingredientUnit < 0)